docs/modules/getting-started/pages/get-started-binary.adoc

```diff
@@ -637,14 +638,14 @@ Go::
 import "github.com/hazelcast/hazelcast-go-client"
 
 func mapSampleRun() {
-  cb := hazelcast.NewConfigBuilder()
+  config := hazelcast.Config{}
 
-  cb.Cluster().SetName("hello-world")
+  config.Cluster.Name = "hello-world"
 
-  client, err := hazelcast.StartNewClientWithConfig(cb) 
+  ctx := context.TODO()
+  client, err := hazelcast.StartNewClientWithConfig(ctx, config)
 
   <1>
-  ctx := context.TODO()
   myMap, err := client.GetMap(ctx, "my-distributed-map")
   if err != nil {
     // handle the error
```

Comment:
We should either `panic(err)` inside if or remove these handlers and add `// error handling is omitted for brevity` warning after line 640.